lib/public/views/Runs/Runs.js

```diff
@@ -196,12 +196,37 @@ export default class Overview extends Observable {
 
     /**
      * Filter runs based on the selected run numbers and fields to prepare it for export
-     * @param {Object} runs The runs data
      * @returns {Object} The Filtered runs data
      */
-    async getFilteredRuns(runs) {
-        const selectedRuns = runs.filter((run) => this.getSelectedRunNumbers().includes(run.runNumber));
-        const filteredRuns = selectedRuns.map((selectedRun) => pick(selectedRun, this.getSelectedRunsFields()));
+    async getFilteredRuns() {
+        this.runs = RemoteData.loading();
+        this.notify();
+
+        const runIds = this.getSelectedRunNumbers().split(',');
+        const foundRuns = [];
+        let response;
```

Comment:
`response` and `result` are used only within the `for` loop. 
I would recommend to use them only within the `for` loop scope.